content/applications/hr/fleet/models.rst

```diff
@@ -0,0 +1,194 @@
+New vehicle models can easily be added to the database. To add a new model, navigate to
+:menuselection:`Fleet app --> Configuration --> Models: Models`. Click :guilabel:`New` and a
+vehicle model form loads. Enter the following information on the form. Be advised, some fields are
+specific to Belgian based companies, so not all fields or sections may be visible depending on the
+location of the company.
+
+.. image:: models/mini.png
+   :align: center
+   :alt: A new model form with the informatoin filled out for a Mini Cooper.
+
+- :guilabel:`Model name`: enter the model name in the field.
+- :guilabel:`Manufacturer`: select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  and edit..`.
+- :guilabel:`Vehicle Type`: select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application since vehicles can be part of an employee's benefits.
+  Adding additional vehicle types is not possible as it affects payroll.
+- :guilabel:`Category`: select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create (new
+  category)`.
+
+.. note::
+   When the manufacturer is selected, the logo for the manufacturer will automatically load in the
+   image box in the top-right corner.
+
+Information tab
+---------------
+
+Model
+~~~~~
+
+- :guilabel:`Seats Number`: enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: enter the number of doors the vehicle has.
+- :guilabel:`Color`: enter the color of the vehicle.
+- :guilabel:`Model Year`: enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: tick this box if the vehicle has a trailer hitch installed.
+
+Salary
+~~~~~~
+
+.. note::
+   The :guilabel:`Salary` section only appears for Belgian-based companies, and only if the company
+   has their localization setting set to Belgium. The cost values are all **monthly** with the
+   exception of the :guilabel:`Catalog Value (VAT Incl.)`.
+
+- :guilabel:`Can be requested`: tick this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: this value is automatically calculated based on Belgian laws and regulations,
+  and cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions`
+  field in the :guilabel:`Engine` section of the vehicle form.
+
+.. important::
+   Modifying the :guilabel:`CO2 Emissions` field will adjust the value in the :guilabel:`CO2 fee`
+   field.
+
+- :guilabel:`Cost (Depreciated)`: enter the monthly cost for the vehicle, which appears in the
```

Comment:
also this is so interesting to learn about how the Belgian employees get a car and how it affects their salary! O:

Reply:
I am still flabbergasted at all the rules and fees and taxes, etc, in Europe for all of this